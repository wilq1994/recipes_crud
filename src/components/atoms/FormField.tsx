import styled from "styled-components";

const FormField = styled.div`
  margin-bottom: 1.4rem;
  &:last-child {
    margin-bottom: 0;
  }

  label {
    color: var(--dark-blue-color);
    text-transform: uppercase;
    font-size: 0.75rem;
    font-weight: 400;
    font-weight: 700;
    letter-spacing: 1px;
    display: block;
    margin-bottom: 0.375rem;
  }

  input,
  textarea {
    font-family: sans-serif;
    font-size: 1rem;
    padding: 0.75em 1em;
    box-sizing: border-box;
    width: 100%;
  }

  textarea {
    height: 140px;
    resize: none;
  }
`;

export default FormField;
