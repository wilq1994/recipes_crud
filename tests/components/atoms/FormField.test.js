import React from "react";
import { shallow } from "enzyme";

import FormField from "components/atoms/FormField";

describe("FormField", () => {
  it("should render and match snapshot", () => {
    const component = shallow(
      <FormField><div>Child</div></FormField>
    );

    expect(component.exists()).toBeTruthy();
    expect(component).toMatchSnapshot();
  });
});
