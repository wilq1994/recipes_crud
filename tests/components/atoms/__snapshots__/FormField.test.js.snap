// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormField should render and match snapshot 1`] = `
<div
  className="sc-bdnylx fqNcxX"
>
  <div>
    Child
  </div>
</div>
`;
